test/e2e/features/basic.feature

```diff
@@ -22,165 +22,69 @@ Feature: Basic test
         And stdout should contain "delete"
         And stdout should contain "status"
         And stdout should contain "Flags:"
-        And stdout should contain
-            """
-            Use "crc [command] --help" for more information about a command.
-            """
 
     @darwin @linux @windows
     Scenario: CRC status
         When executing crc status command fails
-        Then stderr should contain
-            """
-            Machine does not exist. Use 'crc start' to create it
-            """
+        Then stderr should contain "Machine does not exist. Use 'crc start' to create it"
 
     @linux
-    Scenario: CRC setup on Linux
+    Scenario: CRC setup
         When executing "crc setup --check-only" fails
         Then executing crc start command fails
         And executing single crc setup command succeeds
 
-    @linux
-    Scenario: Missing CRC setup
-	Given executing "rm ~/.crc/bin/crc-driver-libvirt" succeeds
-        Then executing "crc setup --check-only" fails
-        And starting CRC with default bundle fails
-	And stderr should contain "Preflight checks failed during `crc start`, please try to run `crc setup` first in case you haven't done so yet"
-	And executing crc setup command succeeds
-
     @darwin
-    Scenario: CRC setup on Mac
+    Scenario: CRC setup
         When executing "crc setup --check-only" fails
         And executing single crc setup command succeeds
-        
+
     @windows
     Scenario: CRC setup on Windows
         When executing single crc setup command succeeds
 
     @darwin @linux @windows
-    Scenario: Request start with monitoring stack
-        When setting config property "enable-cluster-monitoring" to value "true" succeeds
-        And setting config property "memory" to value "14000" succeeds
-        Then starting CRC with default bundle fails
-        And stderr should contain "Too little memory"
-        And setting config property "memory" to value "16000" succeeds
-
-    @darwin @linux @windows
-    Scenario: CRC start
+    Scenario: CRC start usecase
+        # Request start with monitoring stack
+        * setting config property "enable-cluster-monitoring" to value "true" succeeds
+        * setting config property "memory" to value "16000" succeeds
+        Given executing crc setup command succeeds
```

Comment:
Also now we are only testing if user provide correct mem value what about the error it display if user doesn't provide the correct mem like it use to be?

Reply:
there's a separate `config.feature` and its covered there, so should be good i think

Reply:
@anjannath I don't see any `config.feature` file, I think @jsliacan mentioned he will create separate task issue for those missing stuff. so once that in place we should take this PR in and iterate over.

Reply:
`config.feature` exists from before, this was removed in an earlier iteration of the PR but the current PR branch doesn't remove that or am i missing something?

Reply:
Yes, I'll make an issue and it'll go into the config integration test. I don't think it's in the current `config.feature`. It's not the most important testcase though and the minimum we declare should work is 14GiB ([docs](https://access.redhat.com/documentation/en-us/red_hat_openshift_local/2.5/html/getting_started_guide/administrative-tasks_gsg#starting-monitoring_gsg)), so maybe I'll use this value instead of 16000 that we had.

Reply:
I updated the issue with both things @praveenkumar noticed: https://github.com/code-ready/crc/issues/3249 and altered the basic.feature and config.feature a little. I moved the "missing setup" scenario to config.feature (which will be replaced by an integration test) and cleaned up scenarios a little more in `basic.feature` which will stay in e2e.